parser-typechecker/src/Unison/Codebase/Serialization/V0.hs

```diff
@@ -338,6 +356,13 @@ getPattern getA = getWord8 >>= \tag -> case tag of
   7 -> Pattern.As <$> getA <*> getPattern getA
   8 -> Pattern.EffectPure <$> getA <*> getPattern getA
   9 -> Pattern.EffectBind <$> getA <*> getReference <*> getLength <*> getList (getPattern getA) <*> getPattern getA
+  10 -> Pattern.SequenceLiteral <$> getA <*> getList (getPattern getA)
+  11 -> do
+    a <- getA
+    op <- getSeqOp
+    l <- getPattern getA
+    r <- getPattern getA
+    pure $ Pattern.SequenceOp a l op r
```

Comment:
If you switch the order, this would just become 

```haskell
let pa = getPattern getA
in Pattern.SequenceOp <$> getA <*> pa <*> getSeqOp <*> pa
```

Reply:
@pchiusano Ah, I had recommended writing `op` first; alternatively we could change the order of the constructor?

Reply:
Sure, that's fine also.